ANDROID_SETUP.md

```diff
@@ -33,24 +39,40 @@ package replace.me.with.your.package.name;
 
 import be.tramckrijte.workmanager.WorkmanagerPlugin;
 import io.flutter.app.FlutterApplication;
-import io.flutter.plugin.common.PluginRegistry;
+import io.flutter.embedding.android.FlutterEngineConfigurator;
+import io.flutter.embedding.engine.FlutterEngine;
 import io.flutter.plugins.GeneratedPluginRegistrant;
 
-public class App extends FlutterApplication implements PluginRegistry.PluginRegistrantCallback {
-  @Override
-  public void onCreate() {
-    super.onCreate();
-    WorkmanagerPlugin.setPluginRegistrantCallback(this);
-  }
-
-  @Override
-  public void registerWith(PluginRegistry registry) {
-    GeneratedPluginRegistrant.registerWith(registry);
-  }
+public class AppJ extends FlutterApplication implements FlutterEngineConfigurator {
+    @Override
+    public void cleanUpFlutterEngine(FlutterEngine flutterEngine) {
+
+    }
+
+    @Override
+    public void configureFlutterEngine(FlutterEngine flutterEngine) {
```

Comment:
maybe change the order of the methods, always looks weird to have a finalizing method before a creating method